experiments/views.py

```diff
@@ -125,3 +129,20 @@ def post(self, request, *args, **kwargs):
             log = ["form invalid"]
         return render(request, self.template_name, {'form': form,
                                                     'log': log})
+
+
+class DataView(View):
+    """
+    View to serve embeded metabase question
+    """
+
+    def get(self, request, *args, **kwargs):
+        payload = {
+            "resource": { "question": 1 },
+            "params": { "authorized_projects": None },
+            "exp": int((datetime.datetime.now() + datetime.timedelta(minutes = 10)).timestamp()) * 1000 # 10 minute expiration in miliseconds
+        }
+
+        token = jwt.encode(payload, settings.METABASE_SECRET_KEY, algorithm="HS256")
+        iframeUrl = f"{settings.METABASE_SITE_URL}/embed/question/{token.decode('UTF-8')}"
```

Comment:
- variables and functions in Python should be lowercase with underscores, not CamelCase